hawkbit-runtime/docker/docker-compose.yml

```diff
@@ -41,6 +41,8 @@ services:
 
   # ---------------------
   # HawkBit service
+  # username: admin
```

Comment:
IMHO it would be enough to mention the default credentials in the documentation only :)

Reply:
Sure, made the change.

Reply:
Somehow this change was not committed